### PR TITLE
Fix/79126 save organizer order

### PR DESCRIPTION
**Pull request by jesseeproductions**
_Ref:_ [C#79126](https://central.tri.be/issues/79126)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | There must be a space between if and the first parenthesis '(' | [Chooser_Meta_Box.php:95](https://github.com/moderntribe/the-events-calendar/blob/acce6471862e71318895d37f3b3249112588f924/src/Tribe/Linked_Posts/Chooser_Meta_Box.php#L95)
:x: | empty() and isset() must only contain variables (PHP 5.2 compatibility) | [organizer.php:65](https://github.com/moderntribe/the-events-calendar/blob/acce6471862e71318895d37f3b3249112588f924/src/functions/template-tags/organizer.php#L65)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Linked_Posts.php [hunk 1]
```diff
@@ -571,6 +589,34 @@ public function link_post( $target_post_id, $subject_post_id ) {
...
+	 * @param array $current_order an array of the linked post ids being saved
+	 */
+	public function order_linked_posts( $target_post_id, $post_type, $current_order ) {
+		update_post_meta( $target_post_id, $this->get_order_meta_key( $post_type ), $current_order );
+	}
+
+	/**
+	 * Get Linked Post Order
+	 *
+	 * @tbd
```
Comment (bordoni) [inline src/Tribe/Linked_Posts.php]: `@since TBD`

File: src/Tribe/Linked_Posts.php [hunk 2]
```diff
@@ -571,6 +589,34 @@ public function link_post( $target_post_id, $subject_post_id ) {
 		return $linked_posts;
 	}
 
+	/**
+	 * Save Order of Linked Posts
+	 *
+	 * @tbd
```
Comment (bordoni) [inline src/Tribe/Linked_Posts.php]: `@since TBD`

File: src/Tribe/Linked_Posts.php [hunk 3]
```diff
@@ -571,6 +589,34 @@ public function link_post( $target_post_id, $subject_post_id ) {
...
+	 *
+	 * @since TBD
+	 *
+	 * @param int $post_id post id to get meta from
+	 * @param string $meta_key the metakey to get the data
+	 *
+	 * @return mixed
+	 */
+	public function get_order_linked_posts( $post_id, $meta_key ) {
+		return get_post_meta( $post_id, $meta_key, true );
```
Comment (bordoni) [inline src/Tribe/Linked_Posts.php]: If we are passing the `$meta_key`, why not just call the method there?

**Pull request merged**